packages/omg-js-rootchain/src/txUtils.js

```diff
@@ -1,4 +1,4 @@
-async function sendTx (web3, txDetails, privateKey) {
+async function sendTx(web3, txDetails, privateKey) {
```

Comment:
Can you put back in those spaces between function name and `(`?
It's part of the Standard code style that we use. Putting them back will avoid whitespace noise in this and future PRs.

Reply:
ah, yeah I can do that. Must have been my VScode that juts formatted it automatically 🤦‍♂